### PR TITLE
Fixed #34912 -- Fixed size of back links and bookmarklet help in admindocs pages.

### PR DESCRIPTION
**Pull request by specialorange**
Made the text visible in the back links on 4 pages, removing the wrongg css class and attributes when only one css class was present

### CODE REVIEW

File: django/contrib/admindocs/templates/admin_doc/bookmarklets.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 {% block content %}
 
-<p class="help">{% blocktranslate trimmed %}
+<p>{% blocktranslate trimmed %}
```
Comment (felixxm) [inline django/contrib/admindocs/templates/admin_doc/bookmarklets.html:15]: I think we should change this to `quiet` to keep the previous color `color: var(--body-quiet-color)`:
```suggestion
<p class="quiet">{% blocktranslate trimmed %}
```

Comment (felixxm) [reply django/contrib/admindocs/templates/admin_doc/bookmarklets.html:15]: @specialorange Can you take a look? :point_up:

Comment (specialorange) [reply django/contrib/admindocs/templates/admin_doc/bookmarklets.html:15]: done, now to try to figure out how to PR the new files instead

**Comment by felixxm** [comment]
@specialorange Thanks :+1: Welcome aboard :boat:

**Review by felixxm — APPROVED** [review]

**Pull request merged**